### ISSUE TITLE
Filter what we display by time of year

### ISSUE THREAD

**Issue by nikhiltri** [art-institute-of-chicago/browser-extension#45]
Use a [librarian programming calendar](https://programminglibrarian.org/articles/calendar-events-celebrations) to preset filters of what the extension displays, e.g., only display works by Black artists during Black History Month, or only women during Women's History Month.

We should show a link to the filter we're using so they can learn more.

**Comment by dylanirlbeck** [comment]
I think this would be a really cool feature (if you're still interested in having it)! It'd be nice to also have a banner of some sort that indicates if the extension is applying the filter, and if so what the filter is for (e.g. Black History Month). 

I'd be happy to give this a shot if you give me some ideas about the UI and/or any useful API info that will be important to applying the filter. Thanks!

**Comment by IllyaMoskvin** [comment]
Hi @dylanirlbeck! Sorry for missing your reply. It's been a busy couple of months!

We are still interested in this feature, but it's blocked by #19, and it requires us to coordinate with content folks at our institution to add new data to artwork records. We can get the ball rolling on that, but we should get the filter settings infrastructure down first in #19, so that it's easier to make the case for prioritizing the data tagging effort. I'll share our thoughts on that issue there shortly.

> It'd be nice to also have a banner of some sort that indicates if the extension is applying the filter, and if so what the filter is for (e.g. Black History Month).

That's a great idea! I spun that out into a separate issue, #47.
